doc/source/dev/index.rst

```diff
@@ -93,6 +93,15 @@ Here's the short summary, complete TOC links are below:
      changed behavior in docstrings, keeping to the NumPy docstring
      :ref:`standard<howto-document>`.
 
+  * NumPy is a project with a lot of technical debt. While we appreciate
+    efforts to refactor and clean up code, please DO NOT submit a pull request
+    whose entire purpose is to convert formatting to f-strings, or reformat
```

Comment:
Perhaps worth fitting the word `stylistic` in here somewhere. We might also want to mention this paragraph in the section that describes coding style too.
```suggestion
    whose entire purpose is stylistic, such as converting formatting to f-strings, or reformating
```